astroid/helpers.py

```diff
@@ -314,3 +316,77 @@ def object_len(node, context=None):
     raise AstroidTypeError(
         f"'{result_of_len}' object cannot be interpreted as an integer"
     )
+
+
+def literal_eval(node_or_string: Union[str, nodes.NodeNG]) -> Any:
+    """
+    Safely evaluate an expression node or a string containing a Python
+    expression.  The string or node provided may only consist of the following
+    Python literal structures: strings, bytes, numbers, tuples, lists, dicts,
+    sets, booleans, and None.
+
+    Can raise `AstroidSyntaxError` or `ValueError`.
+
+    Adapted from `ast.literal_eval`.
+    """
+    if isinstance(node_or_string, str):
+        _node = builder.parse(node_or_string.lstrip(" \t")).body
+        if len(_node) != 1:
+            raise ValueError(f"expected only one expression, found {len(_node)}")
+        node_or_string = _node[0]
+    if isinstance(node_or_string, nodes.Expr):
+        node_or_string = node_or_string.value
+
+    def _raise_malformed_node(node):
+        msg = "malformed node or string"
+        if lno := getattr(node, "lineno", None):
```

Comment:
We're going to end python 3.6 support soon, but 3.7 will still be used for quite some time.

Reply:
Fixed